tests/Filter/FilterTest.php

```diff
@@ -1,94 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-/*
- * This file is part of the Sonata Project package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\DoctrineORMAdminBundle\Tests\Filter;
-
-use PHPUnit\Framework\TestCase;
-use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
-use Sonata\DoctrineORMAdminBundle\Filter\Filter;
-
```

Comment:
why you removed this file?

Reply:
I think it was because this test wasn't testing code from this bundle, but from AdminBundle.